tests/test_ingestion.py

```
@@ -0,0 +1,34 @@
+
+
+class DummyPipeline:
+    def __init__(self, *args, **kwargs):
+        self.args = args
+        self.kwargs = kwargs
+
+
+def test_get_pipeline_sets_must_attach_to_local_pipeline(monkeypatch):
+    called = {}
+
+    def fake_pipeline(*args, **kwargs):
```

Ruff (0.14.7)

Missing return type annotation for private function `fake_pipeline`

(ANN202)